healthcare/v1beta1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java

```diff
@@ -44,18 +44,19 @@
   private static final JsonFactory JSON_FACTORY = new JacksonFactory();
   private static final NetHttpTransport HTTP_TRANSPORT = new NetHttpTransport();
 
-  public static void fhirResourceGetHistory(String resourceName)
+  public static void fhirResourceGetHistory(String resourceName, String versionId)
       throws IOException, URISyntaxException {
     // String resourceName =
     //    String.format(
     //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "fhir-id");
+    // String versionId = "version-id"
 
```

Comment:
Care to give an example?  (What does a Version-id look like)

Reply:
What's a good example to give when it's just a UUID that the API generates for you? Should I just add "-uuid?"

Reply:
You can just say, provided by the API.  (It didn't appear typed, perhaps it should be)